Resources/Prototypes/_ShitChap/Entities/Weapons/Melee/Nullrod.yml

```
@@ -0,0 +1,466 @@
+      DamageOnUntrainedUse:
+        types:
+          Heat: 20
+      UntrainedUseString: nullrod-pistol-untrained-usage-popup
+    - type: Sprite
+      sprite: _ShitChap/Objects/Weapons/Nullrod/casul.rsi
+      state: icon
+    - type: Clothing
+      sprite: _ShitChap/Objects/Weapons/Nullrod/casul.rsi # Bweh!
+      quickEquip: false
+      slots:
+        - suitStorage
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)